EIPS/eip-3529.md

```diff
@@ -0,0 +1,136 @@
+---
+eip: 3529
+title: Reduction in refunds
+author: Vitalik Buterin (@vbuterin), Martin Swende (@holiman)
+discussions-to: https://ethereum-magicians.org/t/eip-3529-reduction-in-refunds-alternative-to-eip-3298-and-3403-that-better-preserves-existing-clearing-incentives/6097
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-04-22
+---
+
+## Simple Summary
+
+Remove gas refunds for SELFDESTRUCT, and reduce gas refunds for SSTORE to a lower level where the refunds are still substantial, but they are no longer high enough for current "exploits" of the refund mechanism to be viable.
```

Comment:
Recommend shortening this up.  Think email subject line or forum thread title or GitHub PR title.
```suggestion
Remove gas refunds for SELFDESTRUCT, and reduce gas refunds for SSTORE.
```
The latter part of this sentence should be left for the motivation section.